src/mailing/test.js

```diff
@@ -1,10 +1,30 @@
-const sendTestEmail = require('./sender');
+require('dotenv').config();
+const mongoose = require('mongoose');
+const { sendTestEmail } = require('./sender');
+const User = require('../models/User'); 
+const CustomError = require('../errors');
+
 const TEST_RECIPIENT_EMAIL = process.env.TEST_RECIPIENT_EMAIL;
 
+// Connect to the MongoDB database
+mongoose.connect(process.env.MONGO_URL, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+});
+
 async function main() {
   try {
-    const response = await sendTestEmail(TEST_RECIPIENT_EMAIL);
-    console.log('Test email sent successfully:', response);
+    //was trying to add a custom user name from db to the email -needs changes on line 19->  User.findOne({ _id: id })
+    const id = '64d44bc8337399ccf9ad7e4d'
+    const user = await User.findOne({ _id: id }).select('-password');
```

Comment:
So for this test we will always be sending to this particular user right? It may be easier for other members to use this test script if we make this just send to the `TEST_RECIPIENT_EMAIL` user:

```suggestion
    const user = await User.findOne({ email: process.env.TEST_RECIPIENT_EMAIL });
```

(redacting the password doesn't really matter here since this is not data that we're returning in an API to any users)

If you tried this before, it may not have worked because of the above comment about the mongoose connection being async